spec.md

```diff
@@ -1247,7 +1328,7 @@ binding requests.
 
 #### cURL
 ```
-$ curl http://username:password@service-broker-url/v2/service_instances/:instance_id/service_bindings/:binding_id -d '{
+$ curl http://username:password@service-broker-url/v2/service_instances/:instance_id/service_bindings/:binding_id?accepts_incomplete=true -d '{
```

Comment:
it's interesting that the example curl here now includes an optional query param

the provision curl request does not include it.

do we care either way?

Reply:
nevermind, curl examples are the same, was looking at the top level request example